packager/packager.cc

```diff
@@ -623,6 +623,15 @@ ast::ExpressionPtr prependPackageScope(const core::GlobalState &gs, ast::Express
     return scope;
 }
 
+ast::ExpressionPtr prependName(ast::ExpressionPtr scope, core::NameRef prefix) {
+    auto *lastConstLit = &ast::cast_tree_nonnull<ast::UnresolvedConstantLit>(scope);
+    while (auto constLit = ast::cast_tree<ast::UnresolvedConstantLit>(lastConstLit->scope)) {
```

Comment:
Hm, why not just use pointers?

```suggestion
    auto lastConstLit = ast::cast_tree<ast::UnresolvedConstantLit>(scope);
    ENFORCE(lastContLit != nullptr);
    while (auto constLit = ast::cast_tree<ast::UnresolvedConstantLit>(lastConstLit->scope)) {
```

(i try to avoid the address-of `&` operator whenever possible)